spec/spec_helper.rb

```diff
@@ -88,7 +89,11 @@
       ex.run
     end
   end
-
+  
+  config.after do |example|
+    save_and_open_page if example.metadata[:type] == :feature and example.exception.present?
```

Comment:
FYI it only activates in feature tests, where you'd certainly want to see the page. It doesn't activate in unit tests, which represent the overwhelming majority of the app's tests. Does that change your view @equivalentideas ?